src/utilcode/util.cpp

```diff
@@ -1306,10 +1303,8 @@ int GetCurrentProcessCpuCount()
 
 #else // !FEATURE_PAL
 
-    SYSTEM_INFO sysInfo;
-    ::GetSystemInfo(&sysInfo);
-    cCPUs = sysInfo.dwNumberOfProcessors;
-    return sysInfo.dwNumberOfProcessors;
+    cCPUs = PAL_GetCurrentProcessCpuCount();
```

Comment:
I meant just removing the ifdef and keeping only the code that was used for Windows before. The GetProcessAffinityMask that the Windows code uses now works on Unix too. It would also unify the behavior with Windows - the function GetCurrentProcessCpuCount never returns larger number than 64 for Windows. And I have just realized that the GetSystemInfo should return number <= 64 in the dwNumberOfProcessors (it means the same number we get from the GetProcessAffinityMask). The MSDN doc says: 
> dwNumberOfProcessors
The number of logical processors **in the current group**.

A CPU group contains max 64 processors. 
These two changes need to match, since there is an array allocated with size equal to the SYSTEM_INFO.dwNumberOfProcessors and indexed by value obtained from the GetCurrentProcessCpuCount.